name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DeutscheBankPDFExtractor.java

```diff
@@ -140,6 +140,11 @@ private void addDividendeTransaction()
                     t.setShares(asShares(v.get("shares")));
                     t.setSecurity(getOrCreateSecurity(v));
                 })
+                
+                // Ex-Dividenden-Tag 04.12.2014
```

Comment:
// Ex-Dividenden-Tag 04.12.2014
not the same for regEx
`^(Ursprungsland) ([\\w]*)? Ex-Tag (?<exDate>\\d+.\\d+.\\d{4})$`

Reply:
changed